config/locales/ca.yml

```diff
@@ -876,6 +876,7 @@ ca:
         pickup_time_placeholder: "Preparat per (és a dir, data / hora)"
         receival_instructions_placeholder: "Instruccions de recepció"
         add_fee: 'Afegeix una comissió'
+        remove: Eliminar
```

Comment:
we only update en.yml file
all the other files must be left as they are because they will be automatically updated through the transifex process

Reply:
Yes. Remember all this is explained in https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-%28i18n%29

Reply:
argh that's my git revert not reverting properly hold on

Reply:
have pushed again, pls check when you have time :)